contrib/macdeploy/macdeployqtplus

```diff
@@ -543,6 +541,9 @@ ds.close()
 
 # ------------------------------------------------
 
+if platform.system() == "Darwin":
+    subprocess.check_call(f"codesign --deep --force --sign - {target}", shell=True)
```

Comment:
For anyone else's reference, passing `-` to `--sign` invokes ad-hoc signing, which is more limited in it's capabilities, but sufficient for locally running/testing binaries.

> If identity is the single letter "-" (dash), ad-hoc signing is performed.  Ad-hoc signing does not use an identity at all, and identifies exactly one instance of code. Significant restrictions apply to the use of ad-hoc signed code; consult documentation before using this.